docs/Quantiles.ipynb

```diff
@@ -0,0 +1,562 @@
+            "text/plain": [
+              "shape: (1, 1)\n",
+              "┌─────┐\n",
+              "│ AGE │\n",
+              "│ --- │\n",
+              "│ i64 │\n",
+              "╞═════╡\n",
+              "│ 30  │\n",
+              "└─────┘"
+            ]
+          },
+          "execution_count": 7,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "quantile = 0.3\n",
+        "\n",
+        "dp_quantile_30 = context.query().select(\n",
+        "    pl.col(\"AGE\").fill_null(mean_age).dp.quantile(quantile, candidates)\n",
+        ").release().collect()\n",
+        "dp_quantile_30"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bBAzynSmXVbO"
+      },
+      "source": [
+        "### Multiple Quantiles for the Same Variable"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "QCOfRA9_XVbO"
+      },
+      "source": [
+        "To compute multiple quantiles for the same variable, we can use list comprehension to specify multiple quantiles in our query."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 21,
+      "metadata": {},
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "<div><style>\n",
+              ".dataframe > thead > tr,\n",
+              ".dataframe > tbody > tr {\n",
+              "  text-align: right;\n",
+              "  white-space: pre-wrap;\n",
+              "}\n",
+              "</style>\n",
+              "<small>shape: (1, 4)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>Quantile_0.2</th><th>Quantile_0.4</th><th>Quantile_0.6</th><th>Quantile_0.8</th></tr><tr><td>f64</td><td>f64</td><td>f64</td><td>f64</td></tr></thead><tbody><tr><td>20.0</td><td>32.0</td><td>47.0</td><td>65.0</td></tr></tbody></table></div>"
+            ],
+            "text/plain": [
+              "shape: (1, 4)\n",
+              "┌──────────────┬──────────────┬──────────────┬──────────────┐\n",
+              "│ Quantile_0.2 ┆ Quantile_0.4 ┆ Quantile_0.6 ┆ Quantile_0.8 │\n",
+              "│ ---          ┆ ---          ┆ ---          ┆ ---          │\n",
+              "│ f64          ┆ f64          ┆ f64          ┆ f64          │\n",
+              "╞══════════════╪══════════════╪══════════════╪══════════════╡\n",
+              "│ 20.0         ┆ 32.0         ┆ 47.0         ┆ 65.0         │\n",
+              "└──────────────┴──────────────┴──────────────┴──────────────┘"
+            ]
+          },
+          "execution_count": 21,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "multiple_quantiles_actual = df.select(\n",
+        "    [pl.col(\"AGE\").fill_null(mean_age).quantile(q).alias(f\"Quantile_{q}\") for q in [0.2, 0.4, 0.6, 0.8]]\n",
+        ").collect()\n",
+        "multiple_quantiles_actual"
```

Comment:
Is there a reason `release` is needed above but not here?

Also suggest calling it quintile here, and maybe adding a linebreak? something like:
```
quantile(i/5).alias(f\"Quintile_{i}\") for i in range(1, 5)
```